nix/regenerate.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+nix build -f https://github.com/input-output-hk/haskell.nix/archive/master.tar.gz nix-tools --out-link nt
```

Comment:
It's not clear to me which script I need to call to update the nix definitions (`regenerate.sh`, or `update-nixpkgs.sh`, or both) and which files under `nix/` are generated, which are modified, and which are handwritten. Might be worth 
 a few lines in the README.

Reply:
Yes, good point! I really should be better at documentation, given the circumstances...